iotilecore/iotile/core/hw/virtual/virtualdevice_simple.py

```diff
@@ -36,13 +37,15 @@ class SimpleVirtualDevice(BaseVirtualDevice, BaseRunnable):
             for this IOTile device.
         name (str): The 6 byte name that should be returned when anyone asks
             for the controller's name of this IOTile device using an RPC
+        loop (BackgroundEventLoop): The loop we should use for running background
+            tasks. Defaults to the global SharedLoop
     """
 
     __NO_EXTENSION__ = True
 
-    def __init__(self, iotile_id, name):
+    def __init__(self, iotile_id, name, *, loop=SharedLoop):
```

Comment:
Should the "*" argument be part of the Args docstring?

Reply:
I modified the docstring to indicate that `loop` must be passed as a keyword argument.